.github/workflows/revert-tracker.yml

```diff
@@ -25,11 +25,13 @@ jobs:
       - id: generate-reverts-file
         run: |
           cd test-infra/tools
-          python3 -m pip install rockset==1.0.3
+          python3 -m pip install clickhouse-connect==0.7.16
           python3 -m torchci.reverts
           echo "revert_file=$(cat revert_file_name.txt)" >> "${GITHUB_OUTPUT}"
         env:
-          ROCKSET_API_KEY: ${{ secrets.ROCKSET_API_KEY }}
+          CLICKHOUSE_ENDPOINT: ${{ secrets.CLICKHOUSE_HUD_USER_URL }}
+          CLICKHOUSE_USERNAME: ${{ secrets.CLICKHOUSE_HUD_USER_USERNAME }}
```

Comment:
I want to call out that we should have a read-only CH user here, most if not all CI jobs only need read access.  As this is called HUD users, I assume that it is already setup that way?

Reply:
Yes, hud_user has read only permissions